#ifndef FILTER_H
#define FILTER_H

#include "mainwindow.h"
#include <QMainWindow>
#include <opencv2/opencv.hpp>
#include <opencv2/imgproc.hpp>
#include <opencv2/highgui.hpp>
#include <QFileDialog>
#include <QMessageBox>
#include <QPixmap>
#include <QImage>
#include <QVector>
#include <QQueue>
#include <QComboBox>

class filter
{
public:
    filter();
};

#endif // FILTER_H
